index.src.html

```diff
@@ -289,9 +290,8 @@ <h2 id="framework">Framework</h2>
   ISSUE(mikewest/cors-rfc1918#3): What does "private" mean in the context of
   IPv6?
 
-  A <a>request</a> (|request|) is an <dfn export>external request</dfn> if
-  any of the following are true, and an <dfn export>internal request</dfn>
-  otherwise:
+  A <a>request</a> (|request|) is a <dfn export>private network request</dfn>
```

Comment:
How about "cross-network request"?

I find "private network request" to be pretty ambiguous, insofar as it sounds like a request _from_ a private network rather than _into_ a private network. I have two alternative suggestions:

1. "outer network request" would make it clear that the request is coming from "outside" (and we'd need to do some work somewhere in the doc to make it clear that we're thinking about public/private/loopback as something of an onion).

2. "network ingress" (dropping "request" altogether) has some nice connotations as well, and makes it clear that the request is entering the target network from somewhere else.

Both would be somewhat incorrect if we follow the (very reasonable) suggestion to preflight all cross-origin requests for loopback and private hosts. In that case, we might run with something like "unprivileged request" or "network infiltration".

(Sorry to keep painting this bikeshed...)

Reply:
I like network ingress better than private network request. I'd be happy to switch to it. How likely do you think we are to pre-flight all cross-origin requests to local/private? In that case, private network request is pretty accurate.

Reply:
I would answer that question by deferring to the folks who are doing the actual work here (e.g. you :) ).

If you can accept "network ingress", let's just run with that and ignore the fact that it won't be completely accurate in the future.

Reply:
As a non-native speaker, I had to look up the meaning of ingress as it is not a very common word. I think private network request is easier to understand :).

Reply:
Thanks Camille for the input. We definitely want to write specs that are understandable to all developers, regardless of native language. Mike, shall we stick with the current proposal?

Another angle I've been thinking of, but have not been able to map into words: these requests are dangerous in large part because they can leverage the position of the client in the network topology to reach resources they would not otherwise be able to speak to. Something something rebound/star topology/ricochet???

Reply:
Alright. I am leaning towards pre-flighting all cross-origin networks to the private network. In other words, cross-origin private network requests. For now the terminology is not perfect, but I believe it will get much better then.

Reply:
Shall I resolve this conversation?